app/boot/02-add-server-status.js

```diff
@@ -0,0 +1,7 @@
+
+module.exports = function(server) {
+    // Install a `/` route that returns server status
+    const router = server.loopback.Router();
+    router.get('/', server.loopback.status());
```

Comment:
I personally like to use the `/` for documentation. I recommend to use the `/api-status` route for the status

Reply:
That's up for discussion, this is just the result of the default generator.